zulip_bots/zulip_bots/bots/converter/converter.py

```diff
@@ -50,6 +53,61 @@ def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
         bot_response = get_bot_converter_response(message, bot_handler)
         bot_handler.send_reply(message, bot_response)
 
+
+class UnitHandler(object):
+    '''This class handles the units that were input by the user.'''
+
+    def __init__(self, words: List[Any], convert_index: int) -> None:
+        '''Retrieve proper units from utils, convert any abbreviations to full.'''
+        unit_from_raw = words[convert_index + 2]
+        unit_to_raw = words[convert_index + 3]
+
+        self.unit_from = utils.ALIASES.get(unit_from_raw, unit_from_raw)
+        self.unit_to = utils.ALIASES.get(unit_to_raw, unit_to_raw)
+
+    @property
+    def fr(self) -> str:
+        '''Getter for the input unit'''
+        return self.unit_from
+
+    @property
+    def to(self) -> str:
+        '''Getter for the output unit'''
+        return self.unit_to
```

Comment:
We don't really need these getters here, it would just be simpler to access the underlying attributes `unit_from` and `unit_to` directly.